### PR TITLE
[storybook path]

### PR DESCRIPTION
**Pull request by dvdherron**
## Cute animal pic
bild

_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

## Link to Trello card (if applicable)
https://trello.com/c/kWaq6JPM/18-metecho-path-all
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

## Description
_The commit messages say what you did; this should explain why and/or how._
- [x ] Description is in Trello card

## Steps to test/reproduce
yarn storybook 
see the Path story under components
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

## Show me
To come. . .
_Provide screenshots/animated gifs/videos if necessary._

### CODE REVIEW

**Comment by dvdherron** [comment]
[Metecho path › all](https://trello.com/c/kWaq6JPM/18-metecho-path-all)

File: src/stories/components/Path.stories.tsx [hunk 1]
```diff
@@ -32,49 +27,29 @@ const sampleTasks: { [key: string]: Task } = {
...
-  },
+type Props = ComponentProps<typeof Path>;
+interface StoryProps extends Omit<Props, 'task'> {
+  task: Task;
+}
+const TaskTemplate = ({ task, ...rest }: StoryProps) => {
+  return (
+    <TaskStatusPath
+      task={Object.keys(sampleTasks).map((t) => sampleTasks[t])}
+      {...rest}
```
Comment (duggiemitchell) [inline src/stories/components/Path.stories.tsx:38]: @jgerigmeyer Kinda stumped on making the select control work from [this push](https://github.com/oddbird/Metecho/pull/859/commits/45dd2e9af66abdec644588852ef3d8019801ddf3).  Looks like the 'task' prop we are passing is returning an array of Tasks (`Tasks[]`) and I'm not real clear on how to connect the selected task from the control to the component's prop.

Comment (jgerigmeyer) [reply src/stories/components/Path.stories.tsx]: ☑ Fixed in https://github.com/oddbird/Metecho/commit/0622f6d029c488768bc003345e5a62373b88de6a

**Review by stacyk — APPROVED** [review]
LGTM! Nice work!

**Review by jgerigmeyer — APPROVED** [review]

**Pull request merged**